CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.22.2]
+----------
+* feat: Serializes `course-key` field from the federated-content-metadata, CourseDetails model
```

Comment:
nit: `course_key`

nit x2: consider mentioning where the `course_key` is serialized and/or which API so the CHANGELOG is a bit more explicit.